script-library/fish-debian.sh

```diff
@@ -83,9 +83,9 @@ fi
 # Install Fisher
 if [ "${INSTALL_FISHER}" = "true" ]; then
     echo "Installing Fisher..."
-    fish -c 'curl -sL https://git.io/fisher | source && fisher install jorgebucaran/fisher'
+    fish -c 'curl -sL https://raw.githubusercontent.com/jorgebucaran/fisher/HEAD/functions/fisher.fish | source && fisher install jorgebucaran/fisher'
```

Comment:
fisher's READEME uses `main` instead of `HEAD`.
jorgebucaran/fisher#713

Reply:
I just replace it with the actual URL the original `git.io` redirect to.
I can update the link to the one provided in fisher's README if you wish.